Translation/Extractor/File/DefaultPhpFileExtractor.php

```diff
@@ -181,6 +190,19 @@ public function enterNode(Node $node)
     }
 
     /**
+     * @param $node
+     * @return string
+     */
+    private function concatToString($node)
+    {
+        if ($node instanceof String_) {
+            return $node->value;
+        } else {
```

Comment:
Consider the following: 
`$translation->trans('foo'.$bar);`

This will fail. You need to add tests for this and also check if it is an instance of Concat. If not we need to throw an Exception